examples/policies/README.md

```diff
@@ -0,0 +1,16 @@
+### Goal of the 3 policies
+If apply these policies to a managed cluster, it will configure the Prometheus of the managed cluster to forward all it alerts to the alertmanager on the ACM hub. It does so by configuring the cluster-monitoring-config configmap in openshift-monitoring namespace of the managed cluster. This cannot be applied to ACM hub unless [this feature](https://issues.redhat.com/browse/ACM-1756) is delivered.
+
+### Notes
+- When the policies are deleted, we do not delete the resources create by the policies. This is just to play it safe because we are dealing with cluster-monitoring-config configmap - a critical item in configuring OCP monitoring.
```

Comment:
in current logic in endpoint operator, if observability is disabled, we will revert the changes we made before, which is more graceful than the policy

Reply:
We can make that as well by adding one line to all the 3 policies. But if a customer is also adding changes to that cluster-monitoring-config outside our changes (since this is a shared object/API), then we will have problems. So I am having a pause. I am not sure about the best approach here.

Reply:
You can rightly point out that if this configmap is being controlled by policy, as long as policy is enabled, no other changes can be made to the configmap.